extensions/adobe/experience/decisioning/proposition-action.schema.json

```diff
@@ -26,6 +26,14 @@
           "title": "label",
           "description": "The customer generated display name of the element with which the end-user interacted.",
           "type": "string"
+        },
+        "xdm:tokens": {
```

Comment:
Can this field be named a bit more semantically meaningful?

Reply:
Tokens refer to the proposition-interaction tokens. The field "tokens" is enclosed with proposition interaction, just like "label" and "id" that is already present.